argus/action_manager/base.py

```diff
@@ -120,3 +120,30 @@ def is_file(self, path):
     def is_dir(self, path):
         """Check if the directory exists."""
         pass
+
+    @abc.abstractmethod
+    def mkdir(self, path):
+        """Create a directory in the instance if the path is valid.
+
+        :param path:
+            Remote path where the new file should be created.
+        """
+        pass
+
+    @abc.abstractmethod
+    def mkfile(self, path):
+        """Create a file in the instance if the path is valid.
+
+        :param path:
+            Remote path where the new directory should be created.
+        """
+        pass
+
+    @abc.abstractmethod
+    def touch(self, path):
+        """Update the access and modification time to the current time.
```

Comment:
THis description is taking into account the possible flags(`-c`, `-h` and more ), I don't think it is relevant for us.